### PR TITLE
Content/unified doc updates

### PR DESCRIPTION
**Pull request by ddimatos**
This pull request corrects some of the excessive doc its inability to successfully integrate with the unified doc. 

ansible_content.rst is meant only to give the unified doc access to the collections content, not the entire doc site thus it has been scaled back to a more minimal state.

modules.rst had a subheading causing and an unnecessary indentation in the unified doc navigation, with some minor tweaks all the content was preserved and only shuffled in a different order.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: bc62c6046e6ef660aff8ff4267fd73f61b011755
  * Your subject line is longer than 80 characters

Guidelines are available at https://github.com/zhmcclient/zhmc-ansible-modules/blob/master/CONTRIBUTING.rst

---
This message was auto-generated by https://gitcop.com

File: docs/ansible_content.rst [hunk 1]
```diff
@@ -13,122 +13,33 @@
...
+Z HMC
+=====
 
-The **IBM Z HMC collection** provides `Ansible`_ modules that can manage
-platform resources on `IBM Z`_ and `LinuxONE`_ machines.
-
-The goal of this collection is to be able to utilize the power and ease of use
-of Ansible for the management of IBM Z platform resources.
+The **IBM Z® HMC collection**, also represented as
+`ibm_zhmc`_ in this document, is  part of the broader
```
Comment (vkathir82) [inline docs/ansible_content.rst:21]: Need to remove "_" after `ibm_zhmc`

File: docs/ansible_content.rst [hunk 2]
```diff
@@ -13,122 +13,33 @@
...
+initiative to bring Ansible® Automation to IBM Z through the offering
+**Red Hat® Ansible Certified Content for IBM Z**.
 
-The IBM Z platform resources that can be managed include for example partitions,
+The **IBM Z HMC collection** provides `Ansible`_ modules that can manage
```
Comment (vkathir82) [inline docs/ansible_content.rst:25]: _ after the `Ansible`

File: docs/ansible_content.rst [hunk 3]
```diff
@@ -13,122 +13,33 @@
...
+platform resources on `IBM Z`_ and `LinuxONE`_ machines. The IBM Z platform
```
Comment (vkathir82) [inline docs/ansible_content.rst:26]: "_" after the  `IBM Z` and `LinuxONE`

File: docs/ansible_content.rst [hunk 4]
```diff
@@ -13,122 +13,33 @@
...
+resources that can be managed include for example partitions,
 I/O adapters, the Z system itself, or various resources on its Hardware
```
Comment (vkathir82) [inline docs/ansible_content.rst:28]: Require some sentence formatting ... My suggestion : 

"The IBM Z platform resources that can be managed include Partitions, CPU, memory, virtual switches, I/O adapters and various platform resources lifecycle and configuration handled by Hardware Management Console (HMC)."

File: docs/ansible_content.rst [hunk 5]
```diff
@@ -13,122 +13,33 @@
...
-any kind related to this collection (even when they are not about a problem),
-please open an issue in the `issue tracker`_.
+   source/modules
 
-.. _`issue tracker`:
-   https://github.com/zhmcclient/zhmc-ansible-modules/issues
-.. _Apache 2.0 License:
-   https://opensource.org/licenses/Apache-2.0
-.. _ibm.ibm_zhmc collection on Ansible Galaxy:
+.. _ibm_zhmc:
```
Comment (vkathir82) [inline docs/ansible_content.rst:41]: "_" before  ibm_zhmc needs to be removed

File: docs/ansible_content.rst [hunk 6]
```diff
@@ -13,122 +13,33 @@
...
    https://galaxy.ansible.com/ibm/ibm_zhmc/
-.. _zhmc-ansible-modules package on Pypi:
-   https://pypi.org/project/zhmc-ansible-modules/
 .. _Ansible:
```
Comment (vkathir82) [inline docs/ansible_content.rst:43]: "_" before Ansible needs to be removed

File: docs/ansible_content.rst [hunk 7]
```diff
@@ -13,122 +13,33 @@
...
    https://www.ansible.com/
 .. _IBM Z:
```
Comment (vkathir82) [inline docs/ansible_content.rst:45]: Here as well "_" before IBM

File: docs/source/modules.rst [hunk 8]
```diff
@@ -13,30 +13,16 @@
...
-   :maxdepth: 1
-   :glob:
-
-   modules/*
-
-Module Documentation
---------------------
-
 You can also access the documentation of each module from the command line by
 using the `ansible-doc`_ command, for example:
```
Comment (vkathir82) [inline docs/source/modules.rst:27]: "_" after the `ansible-doc` needs to be removed

File: docs/source/modules.rst [hunk 9]
```diff
@@ -46,3 +32,9 @@ using the `ansible-doc`_ command, for example:
 
 .. _ansible-doc:
```
Comment (vkathir82) [inline docs/source/modules.rst:33]: "_" after the ansible-doc word needs to be removed

**Review by vkathir82 — COMMENTED** [review]
I checked these two rst docs in the browser and except minor changes and one sentence correction, things looks good to me

File: docs/ansible_content.rst (hunk 4 above)
Comment (andy-maier) [reply docs/ansible_content.rst:28]: Agreed. I'll update it.

File: docs/ansible_content.rst (hunk 1 above)
Comment (andy-maier) [reply docs/ansible_content.rst:21]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 2 above)
Comment (andy-maier) [reply docs/ansible_content.rst:25]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 3 above)
Comment (andy-maier) [reply docs/ansible_content.rst:26]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 5 above)
Comment (andy-maier) [reply docs/ansible_content.rst:41]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 6 above)
Comment (andy-maier) [reply docs/ansible_content.rst:43]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 7 above)
Comment (andy-maier) [reply docs/ansible_content.rst:45]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/source/modules.rst (hunk 8 above)
Comment (andy-maier) [reply docs/source/modules.rst:27]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/source/modules.rst (hunk 9 above)
Comment (andy-maier) [reply docs/source/modules.rst:33]: Leading and trailing underscores are part of the RST syntax for links. No change needed.

File: docs/ansible_content.rst (hunk 4 above)
Comment (andy-maier) [reply docs/ansible_content.rst:28]: Changed to:
```
The IBM Z platform resources that can be managed include Partitions, CPU,
memory, virtual switches, I/O adapters and various platform resources,
and covers lifecycle and configuration operations.
```

**Comment by andy-maier** [comment]
A copy of the commits plus a commit to resolve review comments is in PR #373 which has been merged. This PR can be closed.

**Pull request closed without merging**